infrastructure/docker/Dockerfile-repository-server

```diff
@@ -0,0 +1,15 @@
+FROM registry.fedoraproject.org/f33/python3
```

Comment:
I'm curious about the use of this image. I usually prefer `alpine` versions like: [3.9.16-alpine3.17](https://github.com/docker-library/python/blob/f568f56f28fab0fe87b34db777e2c2861cef002b/3.9/alpine3.17/Dockerfile).

Reply:
Oh, I didn't know that. I will take a look to that link more in-depth, thanks @akihikokuroda !